tests/snmp/test_snmp_lldp.py

```diff
@@ -34,7 +34,11 @@ def test_snmp_lldp(duthosts, rand_one_dut_hostname, localhost, creds, tbinfo):
     hostip = duthost.host.options['inventory_manager'].get_host(duthost.hostname).vars['ansible_host']
 
     snmp_facts = localhost.snmp_facts(host=hostip, version="v2c", community=creds["snmp_rocommunity"])['ansible_facts']
-    mg_facts   = duthost.get_extended_minigraph_facts(tbinfo)
+    mg_facts = {}
+    for asic_id in duthost.get_asic_ids():
```

Comment:
get_frontend_asic_ids()?

Reply:
SNMP provides LLDP information for all ASICs and not just for front end asics.

Reply:
oh ok. So you also test the backend ASIC as neighbors.

Reply:
yes, that is right.